R/label.R

```diff
@@ -0,0 +1,62 @@
+#' A function to get the labels of data frame columns
+#'
+#' @param data a data frame
+#'
+#' @return labels of the input data frame
+#'
+#' @examples 
+#' get_label(r2rtf::r2rtf_adae)
+#' 
+get_label <- function(data){
+  
+  label <- vapply(data, function(x){
+    if(is.null(attr(x, "label"))){
+      return(NA_character_)
+    }else{
+      attr(x, "label")
+    }
+  }, FUN.VALUE = character(1))
+  
+  ifelse(is.na(label), names(data), label)
+  
+}
+
+#' A function to assign labels to a data frame 
+#'
+#' @param data a data frame
+#' @param var the variables to assign labels
+#' @param label the labels to be assigned
+#'
+#' @return a data frame with labels updated 
+#'
+#' @examples
+#' assign_label(r2rtf::r2rtf_adae)
+#' assign_label(r2rtf::r2rtf_adae, var = "USUBJID", label = "Unique subject id")
```

Comment:
to address CMD issue. 

```suggestion
#' assign_label(r2rtf::r2rtf_adae, var = "USUBJID", label = "Unique subject id")
#' 
#' @export
```